### PR TITLE
IT'S is added

### PR DESCRIPTION
**Pull request by tejasakala**
#21  @suvarna-tammina  IT'S added

### CODE REVIEW

**Review by suvarna-tammina — APPROVED** [review]
@tejasakala Looks Good

**Review by suvarna-tammina — CHANGES_REQUESTED** [review]
@tejasakala  check the example #23 and correct it.

File: app.js [hunk 1]
```diff
@@ -1,2 +1,7 @@
 const { goDucks } = require('./go-ducks');
-console.log(goDucks.quack());
\ No newline at end of file
+<<<<<<< HEAD
+goDucks.appendMessages([() => "IT'S"]);
+console.log(goDucks.quack());
+=======
```
Comment (suvarna-tammina) [inline app.js]: @tejasakala  You didn't merge the code. Please remove the code in line no 2,5. and everything below 5

**Review by suvarna-tammina — CHANGES_REQUESTED** [review]

**Comment by tejasakala** [comment]
@suvarna-tammina made changes, please review

**Review by suvarna-tammina — APPROVED** [review]

**Comment by dhanrajtammina** [comment]
Done by Yamini

**Pull request merged**